static/js/apps/eval_retrieval_generation/sxs/answer_with_tables.tsx

```diff
@@ -31,6 +31,32 @@ interface AnswerWithTablesPropType {
 export function AnswerWithTables(props: AnswerWithTablesPropType): JSX.Element {
   const { sessionQueryId } = useContext(SessionContext);
 
+  const onAnswerChange = () => {
```

Comment:
since we want the same tooltips for the regular RIG eval tool too, I think it'll be easier to just update `processText` util function to take text AND an optional allCalls object & just populate the tooltip there & when doing this, can probably make getting the tooltip html a helper method for readability

Reply:
Right, forgot we wanted the tooltips for regular eval too! Updated both this and the CSS to make tooltips work for both sxs and regular. [Demo](https://github.com/user-attachments/assets/10cac027-2b58-4fa3-8c97-72405c0fb05c)